src/store/flow/flow.ts

```diff
@@ -181,7 +189,7 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
         commit('flow_updateCreatedState', oldCreatedState)
         if(!persistRoute) {
           console.info("Flow persistence route not configured correctly in builder.config.json. Falling back to vuex store")
-          commit('flow_updateFlowContainer', flowContainer)
+          commit('flow_updateFlowContainer', flowContainerCleanedResources)
           return getters.activeFlowContainer 
         }
         return null
```

Comment:
Can we always log something in our catch blocks to avoid silently swallowing unexpected errors?